.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://github.com/m3db/m3/blob/master/CONTRIBUTING.md and developer notes https://github.com/m3db/m3/blob/master/DEVELOPER.md
+2. Please label this pull request according to what type of issue you are addressing.
+3. Ensure you have added or ran the appropriate tests for your PR: read more at https://github.com/m3db/m3/blob/master/DEVELOPER.md#testing-changes
+4. Follow the instructions for writing a changelog note: read more at https://github.com/m3db/m3/blob/master/DEVELOPER.md#adding-a-changelog
+-->
+
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes**:
```

Comment:
I'd say just remove this section and combine it with the `What this PR does / why we need it` section. You can add a comment saying that you should list out the issues that this pr resolves.

Reply:
Sure, combined and kept the issue fixing text.